### DISCUSSION TITLE
Add Get Account API

### DISCUSSION THREAD

**Discussion by nus-will** [vansergen/gemini-node-api#97] [Ideas]
Hi @vansergen ,

I'm using your gemini-node-api, this library is very cool and strong.

I figured out that you didn't cover getAccount api.
Can we integrate this API into your library?
https://docs.gemini.com/rest-api/#account-detail

I knew we provided the getAccounts (https://docs.gemini.com/rest-api/#get-accounts-in-master-group)
But in real case, we also need get the single account.

Should I create a PR for this?

Thank you,

**Comment by vansergen** [comment]
@nus-will Thanks for pointing that out! Please, check out the latest release [v4.0.0](https://github.com/vansergen/gemini-node-api/releases/tag/v4.0.0)
